core-kotlin-modules/core-kotlin-9/src/test/kotlin/com/baeldung/passbyvalue/PassByValueUnitTest.kt

```diff
@@ -0,0 +1,35 @@
+package com.baeldung.passbyvalue
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class PassByValueUnitTest {
```

Comment:
This name implies the unit test only covers passing by value. Can we instead name it PassByValueOrReferenceUnitTest ?